README.md

```diff
@@ -30,173 +35,225 @@ Start by installing the package:
+- `HAYHOOKS_HOST`: The host on which the server will listen.
+- `HAYHOOKS_PORT`: The port on which the server will listen.
+- `HAYHOOKS_PIPELINES_DIR`: The path to the directory containing the pipelines.
+- `HAYHOOKS_ROOT_PATH`: The root path of the server.
+- `HAYHOOKS_ADDITIONAL_PYTHONPATH`: Additional Python path to be added to the Python path.
+- `HAYHOOKS_DISABLE_SSL`: Whether to disable SSL verification when making requests from the CLI.
 
-### Check Hayhooks status
+### CLI commands
 
-From a different shell, you can query the status of the server with:
+The `hayhooks` package provides a CLI to manage the server and the pipelines.
+Any command can be run with `hayhooks <command> --help` to get more information.
 
-```console
-$ hayhooks status
-Hayhooks server is up and running.
+CLI commands are basically wrappers around the HTTP API of the server. The full API reference is available at [//HAYSTACK_HOST:HAYSTACK_PORT/docs](http://HAYSTACK_HOST:HAYSTACK_PORT/docs) or [//HAYSTACK_HOST:HAYSTACK_PORT/redoc](http://HAYSTACK_HOST:HAYSTACK_PORT/redoc).
+
+```shell
+hayhooks run     # Start the server
+hayhooks status  # Check the status of the server and show deployed pipelines
+
+hayhooks pipeline deploy-files <path_to_dir>   # Deploy a pipeline using PipelineWrapper
+hayhooks pipeline deploy <pipeline_name>       # Deploy a pipeline from a YAML file
+hayhooks pipeline undeploy <pipeline_name>     # Undeploy a pipeline
 ```
 
-### Deploy a Haystack pipeline
+### Start Hayhooks
 
-Time to deploy a Haystack pipeline. The pipeline must be in Yaml format (the output of [`pipeline.dump()`](https://docs.haystack.deepset.ai/v2.0/docs/serialization#converting-a-pipeline-to-yaml)), if you don't have one at hand, you can use
-one from this repository. From the root of the repo:
+Let's start Hayhooks:
 
 ```console
-$ hayhooks deploy tests/test_files/test_pipeline_01.yml
-Pipeline successfully deployed with name: test_pipeline_01
+hayhooks run
 ```
 
-Another call to `status` should confirm your pipeline is ready to serve requests:
+This will start the Hayhooks server on `HAYHOOKS_HOST:HAYHOOKS_PORT`.
 
-```console
-$ hayhooks status
-Hayhooks server is up and running.
+### Deploy a pipeline
 
-Pipelines deployed:
-- test_pipeline_01
-```
+Now we will deploy the [Chat with website](https://docs.haystack.deepset.ai/docs/pipeline-templates#chat-with-website) pipeline. We have created an example in the [examples/chat_with_website_streaming](examples/chat_with_website_streaming) folder.
 
-### Have a look at the API schema
+In the example folder, we have two files:
 
-Hayhooks will use introspection to set up the OpenAPI schema accordingly to the inputs and outputs of your pipeline,
-and to see how this works let's get the pipeline diagram with:
+- `chat_with_website.yml`: The pipeline definition in YAML format.
+- `pipeline_wrapper.py`: A pipeline wrapper that uses the pipeline definition.
```

Comment:
Can you explain why we need a pipeline_wrapper ?

Reply:
Good point

Reply:
Added it ;)